integration-tests/tests/api/audit-logs/create-record.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { graphql } from 'testkit/gql';
+import { execute } from 'testkit/graphql';
+import { initSeed } from 'testkit/seed';
+
+test.concurrent('Create Audit Log Record for Organization', async ({ expect }) => {
```

Comment:
TODO @TuvalSimha this file can be merged with the other other file. no need to have two files for two tests